audio-worklet/design-pattern/awn/audio-worklet-processor.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * A simple demonstration of WASM-powered AudioWorkletProcessor.
+ *
+ * @class WASMAudioWorkletProcessor
+ * @extends AudioWorkletProcessor
+ */
+class WASMAudioWorkletProcessor extends AudioWorkletProcessor {
+
+  constructor() {
+    super();
+
+    // Allocate the buffer for the heap access. Start with stereo, but it can
+    // be expanded up to 32 channels.
+    this._heapInputBuffer =
+        new HeapAudioBuffer(WA2.RENDER_QUANTUM_FRAMES, 2, 32);
```

Comment:
I guess the "2" here means stereo?  Maybe add that to the comment

Reply:
The comment says "starts with stereo", but perhaps it wasn't clear enough?